.github/workflows/build-and-publish.yaml

```diff
@@ -115,7 +199,7 @@ jobs:
   check_pages:
     runs-on: ubuntu-latest
     needs: build
-    if: github.event_name == 'pull_request'
+    if: always() && github.event_name == 'pull_request'
```

Comment:
please explain

Reply:
GitHub's job dependency system is weird. Because build depends on monochrome, and check pages depends on build, check pages also implicitly depends on monochrome:

![image](https://github.com/user-attachments/assets/b8c522aa-1e4e-48aa-a14b-4f43860b4ee7)

Therefore, if monochrome was skipped, check pages was also skipped. `always()` is a mechanism to work around that.